datasets/best2009/README.md

```diff
@@ -44,115 +26,109 @@ task_ids:
-```
-
-### Data Fields
+### [Data Instances](#data-instances)
 
-- `fname`: file name; also marks if article is articles, news, encyclopedia or novels
-- `char`: characters
-- `char_type`: character types as adopted from []() by [deepcut](https://github.com/rkcosmos/deepcut)
-- `is_beginning`: is beginning of word
+#### best2009
 
-### Data Splits
+Download size: 13.25 MB
 
-|                         | train      | test    |
-|-------------------------|------------|---------|
-| # lines                 | 148,995    | 2,252   |
-| avg words per line      | 39.05      | NA      |
-| total words             | 5,818,521  | NA      |
-| avg characters per line | 140.39     | 202.79  |
-| total characters        | 20,918,132 | 456,684 |
-| # lines articles        | 16,990     | NA      |
-| # lines encyclopedia    | 50,631     | NA      |
-| # lines novels          | 50,140     | NA      |
-| # lines news            | 31,234     | NA      |
-
-## Dataset Creation
+An example of 'train' looks as follows.
+```
+{
+    "char": ["\u0e08", "\u0e32", "\u0e01", "\u0e15", "\u0e49", "\u0e19", "\u0e09", "\u0e1a", "\u0e31", "\u0e1a", "\u0e40", "\u0e23", "\u0e37", "\u0e48", "\u0e2d", "\u0e07", " ", "\"", "\u0e1a", "\u0e17", "\u0e19", "\u0e33", ":", " ", "\u0e04", "\u0e19", "\u0e08", "\u0e19", "\u0e20", "\u0e32", "\u0e22", "\u0e43", "\u0e15", "\u0e49", "\u0e04", "\u0e27", "\u0e32", "\u0e21", "\u0e2a", "\u0e31", "\u0e21", "\u0e1e", "\u0e31", "\u0e19", "\u0e18", "\u0e4c", "\u0e17", "\u0e32", "\u0e07", "\u0e01", "\u0e0e", "\u0e2b", "\u0e21", "\u0e32", "\u0e22", "\"", "\n"],
```

Comment:
It would be cool to fix the encodings. Not sure how to do it though :/

The example that was already added in the readme was
````
```
{'char': ['?', 'ภ', 'ู', 'ม', 'ิ', 'ป', 'ั', 'ญ', 'ญ', 'า', 'ช', 'า', 'ว', 'บ', '้', 'า', 'น', '\n'], 'char_type': [4, 1, 10, 1, 10, 1, 4, 1, 1, 10, 1, 10, 1, 1, 9, 10, 1, 4], 'fname': 'encyclopedia_00031.txt', 'is_beginning': [1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 1]}
{'char': ['ภ', 'ู', 'ม', 'ิ', 'ป', 'ั', 'ญ', 'ญ', 'า', 'ช', 'า', 'ว', 'บ', '้', 'า', 'น', ' ', 'ห', 'ม', 'า', 'ย', 'ถ', 'ึ', 'ง', ' ', 'ค', 'ว', 'า', 'ม', 'ร', 'ู', '้', 'ข', 'อ', 'ง', 'ช', 'า', 'ว', 'บ', '้', 'า', 'น', ' ', 'ซ', 'ึ', '่', 'ง', 'เ', 'ร', 'ี', 'ย', 'น', 'ร', 'ู', '้', 'ม', 'า', 'จ', 'า', 'ก', 'พ', '่', 'อ', 'แ', 'ม', '่', ' ', 'ป', 'ู', '่', 'ย', '่', 'า', 'ต', 'า', 'ย', 'า', 'ย', ' ', 'ญ', 'า', 'ต', 'ิ', 'พ', 'ี', '่', 'น', '้', 'อ', 'ง', ' ', 'ห', 'ร', 'ื', 'อ', 'ผ', 'ู', '้', 'ม', 'ี', 'ค', 'ว', 'า', 'ม', 'ร', 'ู', '้', 'ใ', 'น', 'ห', 'ม', 'ู', '่', 'บ', '้', 'า', 'น', 'ใ', 'น', 'ท', '้', 'อ', 'ง', 'ถ', 'ิ', '่', 'น', 'ต', '่', 'า', 'ง', 'ๆ', '\n'], 'char_type': [1, 10, 1, 10, 1, 4, 1, 1, 10, 1, 10, 1, 1, 9, 10, 1, 5, 3, 1, 10, 1, 1, 10, 1, 5, 1, 1, 10, 1, 1, 10, 9, 1, 1, 1, 1, 10, 1, 1, 9, 10, 1, 5, 1, 10, 9, 1, 11, 1, 10, 1, 1, 1, 10, 9, 1, 10, 1, 10, 1, 1, 9, 1, 11, 1, 9, 5, 1, 10, 9, 1, 9, 10, 1, 10, 1, 10, 1, 5, 1, 10, 1, 10, 1, 10, 9, 1, 9, 1, 1, 5, 3, 1, 10, 1, 3, 10, 9, 1, 10, 1, 1, 10, 1, 1, 10, 9, 11, 1, 3, 1, 10, 9, 1, 9, 10, 1, 11, 1, 1, 9, 1, 1, 1, 10, 9, 1, 1, 9, 10, 1, 7, 4], 'fname': 'encyclopedia_00031.txt', 'is_beginning': [1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1]}
```
````

Reply:
Trying something there

Reply:
fixed the encoding in the generation app :)